scripts/BUILD.bazel

```diff
@@ -46,3 +46,34 @@ kt_jvm_test(
         "//third_party:org_jetbrains_kotlin_kotlin-test-junit",
     ],
 )
+
+kt_jvm_binary(
+    name = "xml_syntax_check",
+    testonly = True,
+    main_class = "org.oppia.android.scripts.XMLSyntaxCheck$Companion",
+    runtime_deps = [":xml_syntax_check_lib"],
+)
+
+kt_jvm_library(
+    name = "xml_syntax_check_lib",
+    testonly = True,
+    srcs = [
+        "src/java/org/oppia/android/scripts/RepoFile.kt",
+        "src/java/org/oppia/android/scripts/ScriptResultConstants.kt",
+        "src/java/org/oppia/android/scripts/SyntaxErrorHandler.kt",
+        "src/java/org/oppia/android/scripts/XMLSyntaxCheck.kt",
+    ],
+)
+
+kt_jvm_test(
+    name = "test_xml_syntax_check",
```

Comment:
For this & the regex pattern: see other test targets in the codebase to ensure we naming these consistently.

This should be XmlSyntaxCheckTest (to match the name of the test).

/cc @fsharpasharp as well since tests are an intentional deviation in naming strategy so that it's easier to create grouped test commands.

Reply:
Done.